<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainForm</class>
 <widget class="QMainWindow" name="MainForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>417</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>USB Application Example</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>resources/silab.ico</normaloff>resources/silab.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>391</width>
      <height>361</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>391</width>
      <height>361</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>391</width>
      <height>361</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="sysTab">
     <attribute name="title">
      <string>Test</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>361</width>
        <height>80</height>
       </rect>
      </property>
      <property name="title">
       <string>FPGA firmware</string>
      </property>
      <widget class="QLineEdit" name="fileLineEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>231</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="confFPGAButton">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>30</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Conf. FPGA</string>
       </property>
      </widget>
      <widget class="QToolButton" name="fileDialogButton">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>30</y>
         <width>25</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>121</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>GPIO</string>
      </property>
      <widget class="QLineEdit" name="writeDataLine">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>50</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>0x00</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="readDataLine">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>80</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="addLine">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>20</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="writeBtn">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>41</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Write</string>
       </property>
      </widget>
      <widget class="QPushButton" name="readBtn">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>41</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Read</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>409</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>readBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainForm</receiver>
   <slot>readClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>336</x>
     <y>217</y>
    </hint>
    <hint type="destinationlabel">
     <x>342</x>
     <y>432</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>writeBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainForm</receiver>
   <slot>writeClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>359</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>371</x>
     <y>470</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>confFPGAButton</sender>
   <signal>clicked()</signal>
   <receiver>MainForm</receiver>
   <slot>confFPGA()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>423</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fileDialogButton</sender>
   <signal>clicked()</signal>
   <receiver>MainForm</receiver>
   <slot>openFileDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>292</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>461</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>plotData()</slot>
  <slot>openFileDialog()</slot>
  <slot>confFPGA()</slot>
  <slot>writeClicked()</slot>
  <slot>readClicked()</slot>
 </slots>
</ui>
